introduction.md

```diff
@@ -1 +1,7 @@
-Hi, this is christine lam.  My fav code school path is electives.  
+Hi, this is christine lam.  My fav code school path is electives.
+*Git
+*Javascript
+*Java J2SE, J2EE
+*Weblogic
+*JSON...etc
+* adding more changing Git hub is not getting deadly_skills branch 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?